### DISCUSSION TITLE
I alred install KTX-sofware on my Mac but still got error using gltf-transform to run the command

### DISCUSSION THREAD

**Discussion by harmnot** [donmccurdy/glTF-Transform#970] [Q&A] [answered]
I always got this `Command "toktx" not found. Please install KTX-Software` thought I already install them
<img width="1165" alt="image" src="https://github.com/donmccurdy/glTF-Transform/assets/42674439/9bbae839-3716-4e4a-9f17-c49f1e7bac03">

I run this command
```
 gltf-transform etc1s main.glb main-compressed-v1.glb --quality 255 --verbose
```
is that something I missed here ??

u already follow the instruction install the KTX-sofware from their README.md

**Comment by donmccurdy** [comment, marked as answer]
The `toktx` utility will need to be installed and available on the [system path](https://en.wikipedia.org/wiki/PATH_(variable)). If you're on a macOS or Linux system, running `which toktx` (by itself) should tell you where the executable is located, if it is found. If it isn't found, and `toktx` does not run in the CLI, you may need to consult the KTX-Software project for help.
